packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -190,16 +201,42 @@ export const defaultProps: DefaultProps = {
     cookieTechnologiesLink: '',
 };
 
-export const defaultLanguage = new Map<CountryCode, LanguageCode>([
-    [Country.GREAT_BRITAIN, Language.ENGLISH],
-    [Country.FRANCE, Language.FRENCH],
-    [Country.DENMARK, Language.DANISH],
-    [Country.SPAIN, Language.SPANISH],
-    [Country.ITALY, Language.ITALIAN],
-    [Country.NETHERLANDS, Language.DUTCH],
-    [Country.POLAND, Language.POLISH],
-    [Country.SLOVAKIA, Language.SLOVAK],
-    [Country.BULGARIA, Language.BULGARIAN],
-    [Country.GERMANY, Language.GERMAN],
-    [Country.ISRAEL, Language.HEBREW],
+// Available locale files
+export const availableLocales = new Set([
+    `${Language.BULGARIAN}`,
+    `${Language.DANISH}`,
+    `${Language.ENGLISH}-${Country.FRANCE}`,
+    `${Language.FRENCH}-${Country.FRANCE}`,
+    `${Language.FRENCH}`,
+    `${Language.GERMAN}`,
+    `${Language.ENGLISH}`,
+    `${Language.HEBREW}`,
+    `${Language.ITALIAN}`,
+    `${Language.DUTCH}`,
+    `${Language.POLISH}`,
+    `${Language.SLOVAK}`,
+    `${Language.CATALAN}`,
+    `${Language.SPANISH}`,
+]);
+
+// Map the default locale for each Country
+export const defaultLocaleForCountry = new Map<CountryCode, string>([
+    [Country.AUSTRALIA, `${Language.ENGLISH}`],
+    [Country.AUSTRIA, `${Language.GERMAN}`],
+    [Country.BELGIUM, `${Language.FRENCH}`],
+    [Country.BULGARIA, `${Language.BULGARIAN}`],
+    [Country.CANADA, `${Language.ENGLISH}`],
+    [Country.DENMARK, `${Language.DANISH}`],
+    [Country.FRANCE, `${Language.FRENCH}-${Country.FRANCE}`],
+    [Country.GERMANY, `${Language.GERMAN}`],
+    [Country.GREAT_BRITAIN, `${Language.ENGLISH}`],
+    [Country.IRELAND, `${Language.ENGLISH}`],
+    [Country.ISRAEL, `${Language.HEBREW}`],
+    [Country.ITALY, `${Language.ITALIAN}`],
+    [Country.LUXEMBOURG, `${Language.FRENCH}`],
```

Comment:
I think I can see the difference looking through this list – but I do find it a bit strange, that the `Language.FRENCH` default isn't what's used in France 😆

Reply:
Yes, for legal reasons, France require specific wording for their locale file